Error assembling WAR: webxml attribute is required

Hi, 

when I try to execute the demos (e.g. [all-latest-core](https://github.com/jenkinsci/custom-war-packager/tree/master/demo/all-latest-core)) it fails with `Error assembling WAR: webxml attribute is required` (full log below). I tried several Java/Maven version combinations (Java 1.8.0_202, Maven 3.6.0, Maven 3.5.4) without success.

Adding the 'webxml' parameter to the 'maven-war-plugin' config in `tmp\output\pom.xml` and letting it point to `tmp\prebuild\exploded-war\WEB-INF\web.xml` makes it build successfully. I prepared a Pull Request #67.

Question, am I doing something wrong, is my environment wrong, or is this a bug?

Thanks and kind regards, Alex

### Log

```
[INFO] Building war: [...]\custom-war-packager\demo\all-latest-core\tmp\output\target\jenkins-all-latest-1.0-SNAPSHOT.war
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.698 s
[INFO] Finished at: 2019-01-24T12:00:39+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-war-plugin:3.0.0:war (default-war) on project jenkins-all-latest: Error assembling WAR: webxml attribute is required (or pre-existing WEB-INF/web.xml if executing in update mode) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
Exception in thread "main" java.io.IOException: Command failed with exit code 1: cmd.exe /C mvn.cmd clean package
        at io.jenkins.tools.warpackager.lib.util.SystemCommandHelper.processFor(SystemCommandHelper.java:39)
        at io.jenkins.tools.warpackager.lib.util.MavenHelper.run(MavenHelper.java:66)
        at io.jenkins.tools.warpackager.lib.util.MavenHelper.run(MavenHelper.java:46)
        at io.jenkins.tools.warpackager.lib.impl.Builder.build(Builder.java:161)
        at io.jenkins.tools.warpackager.cli.Main.main(Main.java:56)
```

--- a/custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/impl/MavenWARPackagePOMGenerator.java
+++ b/custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/impl/MavenWARPackagePOMGenerator.java
@@ -97,6 +97,7 @@ public class MavenWARPackagePOMGenerator extends POMGenerator {
         mavenHPIPlugin.setGroupId("org.apache.maven.plugins");
         mavenHPIPlugin.setArtifactId("maven-war-plugin");
         mavenHPIPlugin.setVersion("3.0.0");
+        mavenHPIPlugin.setConfiguration(generateMavenWarPluginConfiguration(manifestEntries));
         PluginExecution execution = new PluginExecution();
         execution.setId("package-war");
         execution.addGoal("war");
@@ -114,6 +115,15 @@ public class MavenWARPackagePOMGenerator extends POMGenerator {
         if (value != null) {
             map.put(key, value);
         }
+    }
+
+    private Xpp3Dom generateMavenWarPluginConfiguration(Map<String, String> manifestEntries) {
+        Xpp3Dom webXml = new Xpp3Dom("webXml");
+        webXml.setValue(new File(sourceWar, "WEB-INF/web.xml").getAbsolutePath());
+
+        Xpp3Dom dom = new Xpp3Dom("configuration");
+        dom.addChild(webXml);
+        return dom;
     }
 
     private Xpp3Dom generateCustomWarGoalConfiguration(Map<String, String> manifestEntries) {

